pkg/crane/pull.go

```diff
@@ -35,10 +35,15 @@ func init() { Root.AddCommand(NewCmdPull()) }
 // NewCmdPull creates a new cobra.Command for the pull subcommand.
 func NewCmdPull() *cobra.Command {
 	return &cobra.Command{
-		Use:   "pull",
+		Use:   "crane pull [image] [output-path]",
 		Short: "Pull a remote image by reference and store its contents in a tarball",
-		Args:  cobra.ExactArgs(2),
-		Run:   pull,
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 2 {
```

Comment:
We  also should handle when length of args is 0. So we can use switch statement instead of ifs.

Reply:
Is there a different error message when args equal to 0?